management/src/main/resources/reference.conf

```diff
@@ -55,10 +55,22 @@ akka.management {
     liveness-path = "alive"
     # All readiness checks are executed in parallel and given this long before the check is timed out
     check-timeout = 1s
-    # List of fully qualified class names of readiness checks
-    readiness-checks = ${?akka.management.health-checks.readiness-checks} []
-    # List of fully qualified class names of liveness checks
-    liveness-checks = ${?akka.management.health-checks.liveness-checks} []
+    # Add readiness and liveness checks to the below config objects with the synax:
+    #
+    # name = "FQCN"
+    #
+    # For example:
+    #
+    # cluster-membership = "akka.management.cluster.scaladsl.ClusterMembershipCheck"
+    #
+    # Modules that contribute checks are expected to add their own checks to their reference.conf
```

Comment:
Libraries and frameworks that contribute checks are expected to add their own checks to their reference.conf. Applications can add their own checks to application.conf.